lib/bolt/config.rb

```diff
@@ -127,19 +137,20 @@ def update_from_file(data)
 
       if data['pcp']
         if data['pcp']['service-url']
-          self[:transports][:pcp][:service_url] = data['pcp']['service-url']
+          self[:transports][:pcp][:"service-url"] = data['pcp']['service-url']
         end
         if data['pcp']['cacert']
           self[:transports][:pcp][:cacert] = data['pcp']['cacert']
         end
         if data['pcp']['token-file']
-          self[:transports][:pcp][:token_file] = data['pcp']['token-file']
+          self[:transports][:pcp][:"token-file"] = data['pcp']['token-file']
         end
         if data['pcp']['task-environment']
-          self[:transports][:pcp][:orch_task_environment] = data['pcp']['task-environment']
+          self[:transports][:pcp][:task_environment] = data['pcp']['task-environment']
```

Comment:
Nit: Should this be task-environment for consistency?

Reply:
It doesn't have to be (the others are written so to simplify https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/orch.rb#L15-L25). I can change it easily enough.

Reply:
Done.